app/styles/headers/_second.scss

```diff
@@ -0,0 +1,74 @@
+// styles for second header
+
+.header-second {
+	color: white;
+	padding: 85px 0px 95px 0px;
+	background: #d9c26c;
+	font-family: "Cabin";
+	font-weight: 600;
+	font-size: 16px;
+	height: 372px;
+}
+
+.header-second-content{
+	max-width: 940px;
+	position: relative;
+	margin: 0 auto;
+	height: 456px;
+
+}
+
+.header-second__bars{
+	float: left;
+}
+.header-second__check{
+	display: none;
+}
+
+.header-second__nav{
+	right: 0;
+	position: absolute;
+}
+
+.nav__link{
+	margin-left: 40px;
+	text-decoration: none;
+	color: white;
+}
+
+
+.header-second__logo{
+	position: absolute;
+	left: 50%;
+	transform: translateX(-50%);
+	top: 75px;
+}
+
+.header-second__head{
+	position: absolute;
+	font-size: 36px;
+	line-height: 48px;
+	letter-spacing: 0.72px;
+	text-align: center;
+	top: 180px;
+	width: 100%;
+	font-weight: 400;
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+.header-second__button{
+	text-decoration: none;
+	color: white;
+	border-radius: 24px;
+	background-color: #24aa98;
+	padding: 16px 31px;
+	position: absolute;
+	bottom: 85px;
+	left: 50%;
+	transform: translateX(-50%);
+}
+
+.nav__link:hover, .header-second__button:hover{
+	opacity: 0.8;
+}
```

Comment:
please be attentive with a new line at the end of files

Reply:
ok